extractor.py

```diff
@@ -9,13 +11,16 @@ class Function:
     """
 
     def __init__(self, name, docstring, arg_names, arg_types,
```

Comment:
I think it would be nice to change the order to
```
name
docstring
func_descr
arg_names
arg_types
arg_descrs
return_type
return_expr
return_descr
```

`arg_descr` does not have to be a dictionary. `arg_types` is also just position based

Reply:
I do agree about the order change.
However, the reason why I chose a dictionary is that I think that there are cases where not all parameters have descriptions (because they forgot to add a description for example). Then the descriptions would not match. 
Also, the ordering of parameters in the docstring could be different then the function.

If others think this would not be an issue, than I will gladly change it too a list.

Reply:
Hadn't thought of possible different orderings, the dictionary makes sense in that case :)